students/13th/hyunjungkoo/js/main.js

```diff
@@ -0,0 +1,174 @@
+const searchInput = document.querySelector("#nav .middleContent input");
+const replyInput = document.querySelector(".commentInput input");
+const commentList = document.querySelector(
+  "#main article .comments .commentList"
+);
+const searchBox = document.querySelector("#nav .middleContent .searchBox");
+const friendListBox = document.querySelector(
+  "#nav .middleContent .searchBox .searchList"
+);
+
+const friendList = {
+  wecode_bootcamp: {
+    img_src: "../img/wecode.jpg",
+    name: "위코드|WeCode",
+    href: "https://www.instagram.com/wecode_bootcamp/",
+  },

```

Comment:
```suggestion
const friendList = [{
    user_id: wecode_bootcamp
    img_src: "../img/wecode.jpg",
    name: "위코드|WeCode",
    href: "https://www.instagram.com/wecode_bootcamp/",
  },
```
데이터 형태를 이렇게 구성한다면 어떨까요? 그렇게 하면 굳이 객체를 순회 가능하도록 만들어주지 않아도 반복 가능할 것 같습니다.